backend/corpora/common/utils/uuid.py

```diff
@@ -0,0 +1,8 @@
+import uuid
+
+
+def generate_id():
```

Comment:
I wanted to avoid code repetition. I was also separating out the concern of generating a UUID and converting it into a usable format from the code of creating a new DB row. I think laying it out like this give us room to grow even if we don't use it.

Reply:
this seems more of a stylics change than functional one, but if it bothers you that much, i'll inline it.

Reply:
Stylistic changes are important too, because they have a huge impact on readability, maintainability, debuggability and others. Code reviews are not meant to be only about whether the function performs what it says it performs. If you're interested in learning a bit more on some of the things I look for in code reviews, here's roughly the training I got: https://google.github.io/eng-practices/review/. Functionality is only one of bullet points.

Yes, I'd prefer for it to be inlined. :)